README.md

```diff
@@ -225,8 +225,48 @@ docker build -f x86-openvino.Dockerfile -t "neuralet/smart-social-distancing:lat
 # 2) Run Docker container:
 docker run -it -p HOST_PORT:8000 -v "$PWD":/repo  -e TZ=`./timezone.sh` neuralet/smart-social-distancing:latest-x86_64_openvino
 ```
+## Processor
+
+### Optional Parameters
+
+This is a list of optional parameters for the `docker run` commands.
+They are included in the examples of the [Run the processor](#run-the-processor) section.
+
+#### Logging in the system's timezone
 
-### Configurations
+By default all docker containers use **UTC** as timezone, passing the flag ``` -e TZ=`./timezone.sh` ``` will make the container run on your system's timezone.
+
+You may hardcode a value rather than using the `timezone.sh` script, such as `US/Pacific`. Changing the processor's timezone allows to have better control of when the `reports` are generated and the hours to correlate to the place where the processor is running.
+
+Please note that the bash script may require permissions to execute `chmod +777 timezone.sh`
+
+#### Persisting changes files
```

Comment:
This title sounds wrong. Either "Persisting changed files" or "Persisting changes" (or something else)

BTW, there are a lot of references to this section which need to be updated

Reply:
Done